### ISSUE TITLE
Offsets

### ISSUE THREAD

**Issue by Rorky** [Jire/Charlatano#116]

**Comment by Rorky** [comment]
shit my bad. original comment

why don't you show us how you edit offsets. it is open source.

**Comment by ryleykimmel** [comment]
```
    		BaseEntityOffset = 0xa4c58c
    		EntityArrayOffset = 0x4a6da5c
    		ActionAttackOffset = 0x2ead36c
    		ActionJumpOffset = 0x4f04be0
    		EnginePointerOffset = 0x5a29fc
    		EnginePositionOffset = 0x67e488
    		CvarMouseSensOffset = 0x14ef5
    		CvarYawOffset = 0x14f75
    		CvarMouseEnableOffset = 0x15235
```

Here are the offsets I know of that have changed.

Offset stuff is located [here.](https://github.com/Jire/Charlatano/blob/master/src/main/kotlin/com/charlatano/game/offsets/) However I'm unsure of how @Jire's offset pattern works at the current time.

**Comment by ryleykimmel** [comment]
Netvars:
https://pastebin.com/g87j7e2x
https://pastebin.com/rXZPLiCT

**Comment by Rorky** [comment]
Charlatano/src/main/kotlin/com/charlatano/game/offsets/ClientOffsets.kt 

file path of offsets (since its been the most recently updated and the rest are 2-3 months.)

i know those offsets but these are coded more oddly from what i see.

**Comment by ryleykimmel** [comment]
Yes they are. Like I said above I'm unsure of how they work at the current time.

**Comment by kominxe** [comment]
@Rorky @ryleykimmel file ClientOffsets.kt it's a patter for a offsets-scanner (i think).

**Comment by HoopGit** [comment]
If there's anyone that knows how to update the offsets using Jire's form (I'm not a coder, but I can update offsets the normal way) that'd be great!

**Comment by xyzVyrus** [comment]
As of 5/16/2017, There has been recent changes in the offset patterns for Charlatano due to the newer CS:GO update. Though these were changed, the program still does not work. I am running the program just as I used to when everything worked, and I have made sure that all scripts are completely untouched.

https://gyazo.com/65f8b9277dbcae5b9bc1d4b4ac01dea1 (Changes made)

https://gyazo.com/89784a96ae2e888449a20e2aab8218fc (Error I am receiving)

I'm not _exactly_ sure what an offset is (due to the fact I'm not very familiar with Kotlin) but I was wondering if anyone could be of any assistance in helping me fix the program.

Thank you! (:
